modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/client/util/MeasurementConverterClient.java

```diff
@@ -141,6 +141,21 @@ public static String getMeasurementUnitAbbreviation(MeasurementUnits units) {
         }
     }
 
+    public static MeasurementUnits bestFitUnits(double[] values,  MeasurementUnits originalUnits) {
+        // find bestFit units by taking the average
+        Double average = 0.0;
+        for (int i = 0, sz = values.length; i < sz; i++) {
```

Comment:
Why not just use values.length instead of ``sz`` ? To me at least ``values.length`` is more readable

Reply:
+1, I just copied the source code and made it a function, I'll change.